concent_api/core/tasks.py

```diff
@@ -111,7 +111,10 @@ def call_upload_acknowledged() -> None:
                 result_package_hash=report_computed_task.package_hash,
             )
 
-        transaction.on_commit(call_upload_acknowledged, using='control')
+        transaction.on_commit(
+            call_upload_acknowledged,
+            using='control',
+        )
```

Comment:
Why not keep it on a single line? It was short and readable.

Reply:
I thought we wanted it that way to keep history changes of each parameters, didn't we?